packaging_automation/README.md

```diff
@@ -18,62 +14,137 @@ cd tools
+This flag can be used for testing purposes. If not used, default branch value is used; i.e. for 'citus' 'master, for '
+citus-enterprise' 'enterprise-master'
 
-**--is_test:** If used, branches would not be pushed remote repository and created release branches would be prefixed with 'test'. Default value is False
+**--is_test:** If used, branches would not be pushed remote repository and created release branches would be prefixed
+with 'test'. Default value is False
 
-**--cherry_pick_enabled:** Available only for patch release. If used, --earliest_pr_date flag also should be used.Gets all PR's with 'backport' label created after earliest_pr_date
+**--cherry_pick_enabled:** Available only for patch release. If used, --earliest_pr_date flag also should be used.Gets
+all PR's with 'backport' label created after earliest_pr_date
 
-**--earliest_pr_date:** Used with --cherry-pick-enabled flag. Date format is 'Y.m.d' e.g 2012.01.21. PR's merged after this date would be listed and cherry-picked.
+**--earliest_pr_date:** Used with --cherry-pick-enabled flag. Date format is 'Y.m.d' e.g 2012.01.21. PR's merged after
+this date would be listed and cherry-picked.
 
 **--schema_version:** Available only for patch release. If used, schema version in citus.control file would be updated.
 
-###Example Usage
+### Example Usage
+
+#### Major
 
-####Major
 ``` console
-python -m  packaging_automation.prepare_release --gh_token <your-personal-token> --prj_name citus --prj_ver 10.1.0 
+python -m  packaging_automation.prepare_release --gh_token <your-personal-token> --prj_name citus --prj_ver 10.1.0
 ```
+
 #### Patch
+
 ``` console
 python -m  packaging_automation.prepare_release --gh_token <your-personal-token> --prj_name citus-enterprise --prj_ver 10.0.4 --schema_version 10.0-5
 ```
 
+## Update Docker Usage
+
+Update docker script updates the docker and changelog files in docker repository required for new release of docker
+images after citus/postgres release
+
+## Script Usage
+
+Script can be used for both citus version upgrades and PostgreSQL updates.
+
+### Available flags
+
+**--gh_token:** Personal access token that is authorized to commit docker project. (Required)
+
+**--postgres_version:** Optional value that could be set when new postgres version needs to be set in docker images
+
+**--prj_ver:** Upcoming version to be used for release. should include three level of digits separated by dots, e.g:
+10.0.1
+(Required)
+
+**--is_test:** If used, branches would not be pushed remote repository and PR would not be created (Optional)
+
+### Example Usage
+
+#### Citus Upgrade
+
+``` console
+ python -m  packaging_automation.update_docker --gh_token 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  --prj_ver 10.0.4
+```
+
+#### Citus and PostgreSQL version upgrade
+
+``` console
+ python -m  packaging_automation.update_docker --gh_token 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  --prj_ver 10.0.4 --postgres-version 14.0
```

Comment:
```suggestion
 python -m  packaging_automation.update_docker --gh_token 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  --prj_ver 10.0.4 --postgres_version 14.0
```

Reply:
I realized the invalidated keys in README  even I fixed and commited but missed pushing it :(. I've just pushed it